packages/animations/lib/src/page_transition_switcher.dart

```diff
@@ -203,6 +236,48 @@ class PageTransitionSwitcher extends StatefulWidget {
   /// The child provided to the transitionBuilder may be null.
   final PageTransitionSwitcherTransitionBuilder transitionBuilder;
 
+  /// A function that wraps all of the children that are transitioning out, and
+  /// the [child] that's transitioning in, with a widget that lays all of them
+  /// out. This is called every time this widget is built. The function must not
+  /// return null.
+  ///
+  /// The default [PageTransitionSwitcherLayoutBuilder] used is
+  /// [defaultLayoutBuilder].
+  ///
+  /// The following example shows a [layoutBuilder] that places all entries inside
+  /// a [Column] that sizes its height depending on the heights of active entries.
```

Comment:
Ideally, examples should show good practices. I can't think of any good use cases where putting the children in a column is a good idea. Maybe show an example where a Stack with a different alignment is used (e.g. topleft.)